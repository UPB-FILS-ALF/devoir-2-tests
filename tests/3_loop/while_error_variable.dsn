& while error variable
LOOP: 2
COLOR:    fill 20 20    100
RECTANGLE: 100 100 20   200
CIRCLE: 30 200 200 
ELLIPSE: 100 100 20 
REPEAT