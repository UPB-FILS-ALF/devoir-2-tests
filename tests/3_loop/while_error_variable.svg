<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- loop 2: 2 -->
	<!-- loop #0 -->
		<!-- fill 20, 20, 100 -->
		<polygon points="100,100 20,100 20,200 100,200" stroke="rgb(0,0,0)"  stroke-width="1"  fill="rgb(20,20,100)"/>
		<circle cx="30" cy="200" r="200" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(20,20,100)"/>
		<!-- repeat -->
	<!-- loop #1 -->
		<!-- fill 20, 20, 100 -->
		<polygon points="100,100 20,100 20,200 100,200" stroke="rgb(0,0,0)"  stroke-width="1"  fill="rgb(20,20,100)"/>
		<circle cx="30" cy="200" r="200" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(20,20,100)"/>
		<!-- repeat -->
</svg>
