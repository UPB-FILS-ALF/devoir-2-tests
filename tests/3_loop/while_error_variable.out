ERROR LINE (6): ELLIPSE has 4 parameters, you wrote 3
ERROR LINE (6): ELLIPSE has 4 parameters, you wrote 3
